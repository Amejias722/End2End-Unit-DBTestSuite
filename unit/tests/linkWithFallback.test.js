import { mount } from '@vue/test-utils';

import LinkWithFallBack from '../src/LinkWithFallback.vue';

/**
 * Below is a single test for a UI component. Add additional
 * tests to better validate the functionality of the component.
 */

describe('LinkWithFallback', () => {
  const setupWrapper = function (props) {
    const options = { props };
    const wrapper = mount(LinkWithFallBack, options);
    return wrapper;
  };

  test('Renders plain text with no props', () => {
    const props = {
      link: undefined,
      text: undefined
    };
    const wrapper = setupWrapper(props);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
