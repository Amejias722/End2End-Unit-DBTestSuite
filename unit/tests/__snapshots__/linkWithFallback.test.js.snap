// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LinkWithFallback > Renders plain text with no props 1`] = `<span>no link</span>`;
